jvm-packages/pom.xml

```diff
@@ -37,7 +37,7 @@
         <junit.version>4.13.2</junit.version>
         <spark.version>3.5.1</spark.version>
         <spark.version.gpu>3.5.1</spark.version.gpu>
-        <fasterxml.jackson.version>2.17.2</fasterxml.jackson.version>
+        <fasterxml.jackson.version>2.15.0</fasterxml.jackson.version>
```

Comment:
encountered jackson version issue. so just downgrade it here, will resolve it in the followups